conda_requirements.txt

```diff
@@ -7,3 +7,4 @@ sympy
 numba
 networkx
 h5py
+jsonschema
```

Comment:
We need all the deps to be in the setup.py file.  We add things to the conda file solely to speed up the TravisCI build.

Reply:
Done! A `jsonschema` entry was added to the `install_requires` list.